src/inmanta/app.py

```diff
@@ -72,6 +72,34 @@
 LOGGER = logging.getLogger("inmanta")
 
 
+class MultiLineFormatter(colorlog.ColoredFormatter):
+    """Multi-line formatter."""
+
+    def get_header_length(self, record):
+        """Get the header length of a given record."""
+        header = super().format(
+            logging.LogRecord(
+                name=record.name,
+                level=record.levelno,
+                pathname=record.pathname,
+                lineno=record.lineno,
+                msg="",
+                args=(),
+                exc_info=None,
+            )
+        )
+        len_header = len(header) - 18
+        # 18 is the total length of all the codes used to change the colors in the logs (2 colors "\x1b[3xm" and 2 reset
+        # "\x1b[0m" codes) https://tforgione.fr/posts/ansi-escape-codes/
+        return len_header
+
+    def format(self, record):
+        """Format a record with added indentation."""
+        indent = " " * self.get_header_length(record)
+        head, *trailing = super().format(record).splitlines(True)
```

Comment:
```suggestion
        head, *trailing = super().format(record).splitlines(keepends=True)
```
Prevents people having to go to the docs to see what this is about (like I just did).